core/perftest/src/main/java/edu/uci/ics/texera/perftest/twitter/AsterixTwitterIngest.java

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Making sure that this file didn't come into this PR  by mistake because it isn't related to the bug

Reply:
I suggest we merge this file along with this PR since it's closely related to the twitter feature.